lms/djangoapps/ccx/views.py

```diff
@@ -183,7 +183,21 @@ def create_ccx(request, course, ccx=None):
                 override_field_for_ccx(ccx, vertical, hidden, True)
 
     ccx_id = CCXLocator.from_course_locator(course.id, ccx.id)  # pylint: disable=no-member
+    ccx_course = get_course_by_id(ccx_id)
     url = reverse('ccx_coach_dashboard', kwargs={'course_id': ccx_id})
+
+    # Enroll the coach in the course
+    email_params = get_email_params(course, auto_enroll=True, course_key=ccx_id, display_name=ccx.display_name)
+    enroll_email(
+        course_id=ccx_id,
+        student_email=request.user.email,
+        auto_enroll=True,
+        email_students=True,
+        email_params=email_params,
+    )
+
+    # Grant the coach staff access to the course
+    allow_access(ccx_course, request.user, "staff")
```

Comment:
@sarina with this line the coach will get `staff` access to the new CCX course, not the original one.
The reason is that right now when the CCX is created, the Coach has not privileges to look at the CCX students gradebook or others functionalities needed to administrate the (ccx) course.
BTW: the `staff` has exactly the same meaning in this context.

cc @pdpinch